problemsets-mat223/modules/module11-exercises.tex

```diff
@@ -12,6 +12,28 @@
 			\item $M_{4}=\mat{1&-2&0&-1\\3&5&-1&0\\2&3&-2&0\\0&0&0&1}$.
 		\end{enumerate}
 
+		% Q1 Solutions
+		\begin{solution}
+		    \begin{enumerate}
+    		    \item $M_1 = \mat{1&2&1\\3&1&-2\\8&6&-2}$\\
+    		    $\Null(M_{1})=\Set*{\vec{x} \in \R^3: M_1 \vec{x}=\vec 0}$ \\
+    		    So, we need to solve $\mat{M_1|\vec0}$. And $\rref(M_1)=\mat{1&0&-1\\0&1&1\\0&0&0}$\\
+    		    So, $\Null(M_1)=\Set*{\vec x \in \R^3:\vec x = s\mat{1\\-1\\1} \; \text{for some } s\in\R}= \Span \Set*{\mat{1\\-1\\1}}$\\
```

Comment:
Run these through the formatter. Don't use `\\` for line breaks. Rephrase the sentences so `So` is not used so many times in a row.